control-plane/Dockerfile

```diff
@@ -21,7 +21,7 @@ FROM alpine:3.15 AS dev
 # and the version to download. Example: NAME=consul VERSION=1.2.3.
 ARG BIN_NAME=consul-k8s-control-plane
 ARG VERSION
-ARG ARCH
+ARG TARGETARCH
```

Comment:
Maybe for consistency with the release target, I would add TARGETOS, use it down below in the copy and then maybe set it in the control-plane-dev-docker target.

Reply:
yeah, good idea

Reply:
added!